### ISSUE TITLE
[deeplab] regarding Adding _LOGITS_SCOPE_NAME to exclude_list in utils/train_utils.py

### ISSUE THREAD

**Issue by wenouyang** [tensorflow/models#4375]
In this([issue 4231](https://github.com/tensorflow/models/issues/4231)), the author talks about making modification of the code by 

```
3. Adding _LOGITS_SCOPE_NAME to exclude_list in utils/train_utils.py
# _LOGITS_SCOPE_NAME = 'logits'
exclude_list = ['global_step', 'logits']
if not initialize_last_layer:
exclude_list.extend(last_layers)
```

Under what kind of scenario, we need to make this modification. What does this `logits` stand for?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
